examples/classification_example/cnn.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This file predict the ner tag for conll03 dataset."""
+
+import torch
+from texar.torch.modules.embedders import WordEmbedder
+from texar.torch.modules.classifiers.conv_classifiers import Conv1DClassifier
+from torch import nn
+from texar.torch.data import Batch
+
+
+def pad_each_bach(word, max_sen_len):
```

Comment:
I see you have the utility function now, can you remove this?

Reply:
You wrote this function twice. Can you remove one of them?

Reply:
Removed.